components/GameTable.tsx

```diff
@@ -0,0 +1,115 @@
+"use client";
+import React from "react";
+import { Button, Box } from "@mui/material";
+import { DataGrid, GridColDef, GridRowSelectionModel } from "@mui/x-data-grid";
+import { useEffect, useState } from "react";
+import { GameFormValues } from "./types";
+
+const columns: GridColDef<GameFormValues>[] = [
+  { field: "id", headerName: "ID", width: 50 },
+  {
+    field: "game_name",
+    headerName: "Game Name",
+    width: 250,
```

Comment:
just wondering.. is the table responsive? seems that every column has fixed value in pixels.